2022/ideas-list.md

```diff
@@ -0,0 +1,8 @@
+# Ideas pages for 2022
+
+This is the home page of projects ideas of IOOS for Google Summer of Code 2022.
+
+https://github.com/ioos/gsoc 
+Pyobis (if we can commit to the repo)
+ioos_qc
+IOOS compliance checker
```

Comment:
@benjwadams are you or anyone in your group interested in creating an ideas page and mentoring this year?